src/api/renderer.h

```diff
@@ -205,7 +205,6 @@ class TESS_API TessPDFRenderer : public TessResultRenderer {
   GenericVector<long int> pages_;    // object number for every /Page object
   std::string datadir_;              // where to find the custom font
   bool textonly_;                    // skip images if set
-  int jpg_quality_;                  // set JPEG quality
```

Comment:
Travis failed because of this change.

You should also change the .cpp file.


https://github.com/tesseract-ocr/tesseract/blob/cbd09de7feca46543289d01b2d8482312ced2dd7/src/api/pdfrenderer.cpp#L188

Reply:
@zdenop tried a different solution in commit cbd09de7feca46543289d01b2d8482312ced2dd7. I have to revert that commit.

Reply:
Done now. The updated pull request should pass the CI tests.

Reply:
What he did was right (based on my suggestion).

He just forget to remove the part you now removed.

Reply:
thank stweil. I forget we do not need here.